cardano-cli/golden/Test/Golden/Byron/Vote.hs

```diff
@@ -21,9 +21,9 @@ import           Test.OptParse
 
 golden_byron_yes_vote :: Property
 golden_byron_yes_vote = propertyOnce $ H.moduleWorkspace "tmp" $ \tempDir -> do
-  goldenYesVote <- noteInputFile "test/data/golden/byron/votes/vote-yes"
-  proposal <- noteInputFile "test/data/golden/byron/update-proposal"
-  signingKey <- noteInputFile "test/data/golden/byron/keys/byron.skey"
+  goldenYesVote <- noteInputFile "golden/files/golden/byron/votes/vote-yes"
```

Comment:
isn't that extra `golden` dir redundant? why not just `golden/files/byron/...`

Reply:
The first `golden` is home for the library component for golden tests.  The second golden is home for the golden files.

This leaves space for files that aren't golden files.  For example: `golden/files/data`.  At the moment we throw all the files in the same location regardless of whether they are golden files or not.